docs/getting-started/android/beta-deployment.md

```diff
@@ -2,7 +2,7 @@
 
 # Deploy to Beta distribution services using _fastlane_
 
-If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](setup/#setting-up-supply) before continuing.
+If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](../setup/#setting-up-supply) before continuing.
```

Comment:
Please change this to
```suggestion
If you would like to distribute your beta builds to Google Play, please make sure you've done the steps from [Setting up _supply_](/getting-started/android/setup/#setting-up-supply) before continuing.
```
so it's consistent with the rest of the codebase (we don't use relative URLs at all).